packages/@adobe/spectrum-css-temp/components/dropdown/skin.css

```diff
@@ -45,6 +45,9 @@ governing permissions and limitations under the License.
         color: var(--spectrum-dropdown-icon-color-disabled);
       }
     }
+    &:active .spectrum-Dropdown-label {
+      color: var(--spectrum-dropdown-text-color-disabled);
```

Comment:
I couldn't find this value anywhere else in the code, but it seems to have a fallback and be valid.